src/axom/quest/tests/quest_c2c_reader.cpp

```diff
@@ -32,10 +32,10 @@ namespace quest = axom::quest;
 
 namespace
 {
-static const std::string C2C_LINE_FILENAME = "test_line.contour";
-static const std::string C2C_CIRCLE_FILENAME = "test_circle.contour";
-static const std::string C2C_SQUARE_FILENAME = "test_square.contour";
-static const std::string C2C_SPLINE_FILENAME = "test_spline.contour";
+const std::string C2C_LINE_FILENAME = "test_line.contour";
```

Comment:
`constexpr` for these?

Reply:
:) I tried that too and got an error about `constexpr` and strings.

Looks like we'll have to wait for `std::string_view` in C++17 -- https://stackoverflow.com/questions/27123306/is-it-possible-to-use-stdstring-in-a-constant-expression